tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -81,6 +80,14 @@ public class EnvironmentHelper
 
         public string FullSampleName => $"{_appNamePrepend}{SampleName}";
 
+        public bool UseNativeLoader
+        {
+            get
+            {
+                return string.Equals("true", Environment.GetEnvironmentVariable("USE_NATIVE_LOADER"), StringComparison.InvariantCultureIgnoreCase);
+            }
+        }
+
```

Comment:
Line returns aren't free you know 😉  
```suggestion
        public bool UseNativeLoader => string.Equals("true", Environment.GetEnvironmentVariable("USE_NATIVE_LOADER"), StringComparison.InvariantCultureIgnoreCase);
```